proposals/gc/Overview.md

```diff
@@ -1 +1,716 @@
+# GC Extension
```

Comment:
Not sure myself anymore either. Somehow I probably misread something, and started thinking "JIT". Sorry.